inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -412,7 +415,7 @@ void MKLDNNConvolutionNode::initSupportedPrimitiveDescriptors() {
     // attr[0] - depthwise, quantize
     // attr[1] - binary
     mkldnn::primitive_attr attrs[1];
-    setPostOps(attrs[0]);
+    setPostOps(attrs[0], MemoryDescUtils::makeDummyShape(getOutputShapeAtPort(0)).getStaticDims());
 //    setPostOps(attrs[1], false, true);
```

Comment:
I think we need to update commented option of setPostOps as well.

Reply:
Done